docs/concepts/storage/persistent-volumes.md

```diff
@@ -695,6 +695,8 @@ parameters:
 
 #### Azure Disk
 
+##### Azure Unmanaged Disk Storage Class (before v1.7.2)
```

Comment:
@seanknox I have added the notes in the doc, see:
Premium VM can attach both Standard_LRS and Premium_LRS disks, while Standard VM can only attach Standard_LRS disks.
Managed VM can only attach managed disks and unmanaged VM can only attach unmanaged disks.